UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/NoticestoMariners/Resellers.cshtml

```diff
@@ -0,0 +1,203 @@
+                </tr>
+                <tr>
+                    <td>Hong Yunn Sea Professional Company Limited</td>
+					<td><a href="https://www.hongyunn.com.tw" target="_blank">www.hongyunn.com.tw</a></td>
+                </tr>
+                <tr>
+                    <td>Maritime Services</td>
+					<td><a href="https://www.maritimeservices.ca" target="_blank">www.maritimeservices.ca</a></td>
+                </tr>
+                <tr>
+                    <td>Maryland Nautical Sales Inc.</td>
+					<td><a href="https://www.mdnautical.com" target="_blank">www.mdnautical.com</a></td>
+                </tr>
+                <tr>
+                    <td>Millenium Marine Services EPP</td>
+					<td><a href="https://www.milleniummarine.com" target="_blank">www.milleniummarine.com</a></td>
+                </tr>
+                <tr>
+                    <td>Motion Smith</td>
+					<td><a href="https://www.motsmith.com" target="_blank">www.motsmith.com</a></td>
+                </tr>
+                <tr>
+                    <td>NautoShark</td>
+					<td><a href="https://www.navareas.com" target="_blank">www.navareas.com</a></td>
+                </tr>
+                <tr>
+                    <td>NAVICO Norway AS (C-MAP Update Service)</td>
+					<td><a href="https://www.navico.com" target="_blank">www.navico.com</a></td>
+                </tr>
+                <tr>
+                    <td>Navtor AS</td>
+					<td><a href="https://www.navtor.com" target="_blank">www.navtor.com</a></td>
+                </tr>
+                <tr>
+                    <td>Novaco</td>
+					<td><a href="https://www.novaco.co.uk" target="_blank">www.novaco.co.uk</a></td>
+                </tr>
+                <tr>
+                    <td>OneOcean (Canada) Inc</td>
+					<td><a href="https://www.oneocean.com" target="_blank">www.oneocean.com</a></td>
+                </tr>
+                <tr>
+                    <td>OneOcean Group Limited</td>
+					<td><a href="https://www.oneocean.com" target="_blank">www.oneocean.com</a></td>
+                </tr>
+                <tr>
+                    <td>Plymouth Sea Chest Limited</td>
+					<td><a href="https://www.seachest.co.uk" target="_blank">www.seachest.co.uk</a></td>
+                </tr>
+                <tr>
+                    <td>Poseidon Navigation Services Ltd</td>
+					<td><a href="https://www.poseidonnavigation.com" target="_blank">www.poseidonnavigation.com</a></td>
+                </tr>
+                <tr>
+                    <td>Qija Voyage Ltd</td>
+					<td><a href="http://www.qjvoy.net/en/" target="_blank">www.qjvoy.net</a></td>
+                </tr>
+                <tr>
+                    <td>SEEKARTE Kapitän A. Dammeyer GmbH & Co. KG</td>
+					<td><a href="https://www.seekarte.de" target="_blank">www.seekarte.de</a></td>
+                </tr>
+                <tr>
+                    <td>Shanghai Ocean Nautical Supply Company</td>
+                    <td>---</td>
```

Comment:
Same as above comment. Existing MSI application displaying N/A for website for this VAR. Not sure if we keep it same?
```suggestion
                    <td>N/A</td>
```